craft_providers/lxd/remotes.py

```diff
@@ -75,10 +75,18 @@ def is_stable(self) -> bool:
 
         :returns: True if the image is stable.
         """
-        return (
+        if (
             self.remote_name == BUILDD_RELEASES_REMOTE_NAME
             and self.remote_address == BUILDD_RELEASES_REMOTE_ADDRESS
-        )
+        ):
+            # Ubuntu official buildd images
+            return True
+
+        if self.remote_name == "images":
+            # LXD daily images
```

Comment:
Is this comment accurate?  Or is it about CentOS?

Either way, I think you could drop this if-statement since the function already returns false.